src/storage/v3/shard.cpp

```diff
@@ -1065,8 +1065,10 @@ const std::string &Shard::EdgeTypeToName(EdgeTypeId edge_type) const {
 }
 
 bool Shard::CreateIndex(LabelId label, const std::optional<uint64_t> desired_commit_timestamp) {
-  // TODO Fix Index
-  return false;
+  // Do label indexes make sense on primary labels?
```

Comment:
No, because the shards are the label indices over primary labels.

Reply:
Will add that check as well then